lms/views/decorators/__init__.py

```diff
@@ -0,0 +1,11 @@
+# -*- coding: utf-8 -*-
+
+"""Python decorators meant to be used for decorating Pyramid view functions."""
```

Comment:
My plan is to turn this, and also the LMS app's existing Python decorators (in `lms/util/*.py`) that're used to decorate views, into Pyramid view decorators (as in the [`decorator` view config argument](https://docs.pylonsproject.org/projects/pyramid/en/latest/narr/viewconfig.html#non-predicate-arguments) because decorating views with normal Python decorators is a bad idea for a bunch of reasons. When I do that I'll move all the other view decorators out of `util` and into `views.decorators` too.